library/Icingadb/Widget/ItemList/BaseCommentListItem.php

```diff
@@ -101,7 +102,11 @@ protected function assembleVisual(BaseHtmlElement $visual)
 
     protected function createTimestamp()
     {
-        return new TimeAgo($this->item->entry_time);
+        if ($this->item->expire_time != 0) {
+            return (new TimeUntil($this->item->expire_time))->prepend('expires ');
```

Comment:
Please mark the message translatable by the use of a `FormattedString`.

`FormattedString::create(t('expires %s'), new TimeUntil...`